db/data_migrations/20220805304709_fix_broken_comm_code_descriptions.rb

```diff
@@ -0,0 +1,10 @@
+Sequel.migration do
+  up do
+    tunas = GoodsNomenclatureDescription.where(goods_nomenclature_description_period_sid: 154_585)
+    tunas.update(description: 'Tunas (of the genus Thunnus), skipjack tuna (stripe-bellied bonito) (Katsuwonus pelamis), excluding edible fish offal of subheadings 0302 91 to 0302 99')
```

Comment:
this should be fine,
please could you double-check that there is not any issue in case `tunas` is empty?